packages/api/lambdas/sf-event-sqs-to-db-records/write-execution.js

```diff
@@ -155,8 +158,11 @@ const writeExecution = async ({
       executionModel,
       esClient,
     });
-    return executionCumulusId;
+    await publishExecutionSnsMessage(executionApiRecord);
```

Comment:
So this approach isn't quite the same as what we have now with the DynamoDb streams and Lambdas publishing to the SNS topic. This approach would publish the `executionApiRecord` that we're generating from the Cumulus message to the SNS topic. However, the DynamoDb stream approach publishes the record **as created/updated in DynamoDb** to the SNS topic.

Because of conditional logic in our Dynamo/PG write methods, there may be subtle differences between these two things. For example, when an execution has already been saved to the DB as `completed` and then we process a delayed event in this Lambda where the status of the execution is `running`, the `executionApiRecord` would for the delayed event would have a status of `running`, but the record **would not** get updated from `completed` to `running` in DynamoDB. So sending over the `executionApiRecord` would be inaccurate.

We may need to have an impromptu architecture meeting on the best way to handle this

Reply:
We could possibly modify the PG upsert to return all the properties of the updated record, not just the `cumulus_id`. Then we could translate the PG record to the API structure and send that to the SNS topic. It also isolates us from dependency on DynamoDB, making it easier to remove later